src/main/java/core/basesyntax/model/User.java

```diff
@@ -3,42 +3,23 @@
 import java.util.Objects;
 
 public class User {
-    private Long id;
-    private String login;
+    final String login;
```

Comment:
The `login` field is declared as `final`, which means it must be initialized in every constructor. This is correctly done, but be aware that it cannot be changed after the object is created.